src/high-level/include/iDynTree/KinDynComputations.h

```diff
@@ -552,6 +552,11 @@ class KinDynComputations {
      */
      Vector3 getCenterOfMassBiasAcc();
 
+     /**
+     * Return the linear and angular momentum bias acceleration.
```

Comment:
If we could add a latex math description of what this is it would be great. I know that the rest of the class does not have this  good docs (my fault mainly) but for the future it would be nice to have sane documentation in place. To embed latex in Doxygen blocks, you can use the syntax in http://www.doxygen.nl/manual/formulas.html .